exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -106,6 +106,7 @@
   <p>STATE: ${model.getProfile().getState().name()}</p>
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
+  <p>DURATION: ${model.getProfileDuration()}</p>
```

Comment:
For running queries, I see incorrect duration calculation. On `/profiles` elapsed time is shown correctly but when you step into running profile, duration is shown incorrectly.
![image](https://cloud.githubusercontent.com/assets/15086720/21849179/259dc0ae-d80d-11e6-990d-cbb9e2d072ad.png)
![image](https://cloud.githubusercontent.com/assets/15086720/21849198/3c26d9f0-d80d-11e6-873d-8bd439cc4351.png)